### ISSUE TITLE
Items not backported yet

### ISSUE THREAD

**Issue by ghost** [libretro/Mesen#35]
1. Replace S5b emu with emu2149 for better accuracy
https://github.com/NovaSquirrel/Mesen-X/commits/master/Core/SSGAudio.h
(*) Might make emu much slower on low-power devices?

2. Replace Vrc7 emu with Nuked-OPLL
https://github.com/NovaSquirrel/Mesen-X/commits/e6d75fcf58da2ccc1b89ed1d3c99f74925f2907e/Core/Vrc7Audio.h
(*) Might make emu much slower on low-power devices?

3. Ntsc filter fixes
https://github.com/NovaSquirrel/Mesen-X/commits/master/Core/BisqwitNtscFilter.cpp

4. HD updates (many)

**Comment by ghost** [comment]
I noticed Mesen2 has a lot of new fixes
https://github.com/SourMesen/Mesen2/commits/master/Core/NES


Possible to backport or re-fork the NES core?

**Comment by nfp0** [comment]
Was https://github.com/libretro/Mesen/pull/34 related to this?

**Comment by ghost** [comment]
No. NovaSquirrel/Mesen-X has lots of mapper fixes not here = #34. Post 1 has more Mesen-X changes not in libretro. And Mesen2 has newer NES fixes not seen in any Mesen1 port, which I think can be cherry-picked back here.

**Comment by nfp0** [comment]
Alright, thanks for the clarification.

Hopeful that we'll get the libretro version of Mesen2 one day.

**Comment by ghost** [comment]
Sour said he will create libretro cores for each Mesen2 system but it's a long time off. And if it runs slower than Mesen1, some of us have to stick with older Mesen core.

**Comment by nfp0** [comment]
Good to know he's considering the libretro cores.

But why does it run slower? I thought Mesen2 was just the continued development of the original Mesen, or is it a rewrite?

**Comment by Sanaki** [comment]
Mesen is a high-accuracy emulator. The higher the accuracy, often the slower the emulation.

**Comment by negativeExponent** [comment]
copy/paste much?

**Comment by nfp0** [comment]
> Mesen is a high-accuracy emulator. The higher the accuracy, often the slower the emulation.

Ah yes, that makes perfect sense.
